action.yml

```diff
@@ -12,14 +12,13 @@ runs:
     - name: check OS 
       run: |
         if [ "$RUNNER_OS" != "Linux" ]; then
-          echo "Not supported OS. This action only supports Ubuntu."
+          echo "Not supported OS. This action only supports Linux."
```

Comment:
supported OS is still Ubuntu, because apt install is Ubuntu only

Reply:
done.